.github/workflows/release.yml

```
@@ -133,6 +133,11 @@ jobs:
             arch=$(basename "$apk" | sed -e 's/app-fdroid-//' -e 's/-release.apk//')
             cp "$apk" "build/outputs/Updatium-${{ steps.version.outputs.version }}-fdroid-${arch}.apk"
           done
+        
+      - name: Attest Build Provenance
+        uses: actions//attest-build-provenance@main
```

actionlint (1.7.12)

[error] specifying action "actions//attest-build-provenance@main" in invalid format because owner and repo and ref should not be empty. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)